akka-diagnostics/src/test/scala/akka/diagnostics/AkkaSpec.scala

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
would be good to update headers, by enabling the header plugin (but can be a separate follow up PR if you prefer)